src/compositor/seat/seat.c

```diff
@@ -114,8 +114,10 @@ seat_handle_key(struct wlc_seat *seat, const struct wlc_input_event *ev)
       return;
 
    wlc_keyboard_update_modifiers(&seat->keyboard, ev->device);
+   struct wlc_modifiers mods = {0, 0};
+   uint32_t keysym = wlc_keyboard_get_keysym_for_key_ptr(&seat->keyboard, ev->key.code, &mods);
 
-   if (seat->keyboard.modifiers.mods == (WLC_BIT_MOD_CTRL | WLC_BIT_MOD_ALT) && ev->key.code >= 59 && ev->key.code <= 88) {
+   if (seat->keyboard.modifiers.mods == (WLC_BIT_MOD_CTRL | WLC_BIT_MOD_ALT) && keysym >= XKB_KEY_F1 && keysym <= XKB_KEY_F12) {
```

Comment:
Looking at the keysym constants they seem to rise linearly, I'm fine with this. I doubt that they'll change. There may be problem if somebody provides XKB implementation that has different constant that for some reason doesn't work this way. But if that ever happens lets worry only then. This is more straightforward so I like it.